packages/client-react-streaming/src/DataTransportAbstraction/WrappedApolloClient.test.tsx

```diff
@@ -313,6 +315,7 @@ describe(
         appendToBody`<div hidden id="S:1"><div id="user">User</div></div>`;
         $RS("S:1", "P:1");
 
+        await new Promise((resolve) => setTimeout(resolve, 10));
```

Comment:
Arriving request data (`simulateRequestData`) now takes a tick as it goes through the link chain instead of directly being written to the store. Without this delay, this test would have written the query first and then the data from the link would have arrived.